Assets/Scripts/Road/RoadTrack.cs

```diff
@@ -19,6 +22,29 @@ private void Start()
         }
     }
 
+    /// <summary>
+    /// Stops spawning of cars by killing all coroutines
+    /// </summary>
+    public void DisableSpawnPoints()
+    {
+        StopAllCoroutines();
+    }
+
+    /// <summary>
+    /// Explodes all cars this track has spawned
+    /// </summary>
+    public void BoomAllCars()
+    {
+        foreach (var obst in _spawnedObstacles)
+        {
+            // TODO: find a better way maybe? not that performance is an issue...
+            if(obst != null)
```

Comment:
Vad för du för fel när du kör ReferenceEquals?

Reply:
Får nåt med att "object ur trying to access has been deleted" eller is null eller nåt sånt